controller.js

```diff
@@ -13,7 +13,29 @@ function getTopics(req, res, next){
     .catch(next)
     }
 
-function getArticle(req, res, next){
+    function getArticleComments(req, res,next){
+    const articleId = req.params.article_id
+    if(isNaN(+articleId)){
+    res.status(400).send({msg: "Bad Request"})
+    }
+    
+    returnArticleComments(articleId)
+    .then(comments=>{
+            res.status(200).send({comments})
+        
+    })
+    .catch((error)=>{
+        if(error.status === 404){
+            res.status(404).send({msg : 'Not Found'})
```

Comment:
Hi Stephen, as per Hev's earlier feedback, can this please be moved into error handling middleware? If you need a hand with how this should look, this page is really useful 👍 
https://notes.northcoders.com/courses/js-back-end/error-handling